adam/experiment/experiment_utils.py

```diff
@@ -459,6 +459,11 @@ def build_affordance_learner_factory(
             language_mode=language_mode,
             min_continuous_feature_match_score=min_continuous_feature_match_score,
         )
+    elif learner_type == "mapping":
```

Comment:
So you can't use both this learner and the pattern-based affordance learner?

Reply:
You can, I've noticed that this push doesn't have the added logic to handle loading this second type of affordance learner at the same time. I have that fixed on my local branch.